Quadriga/src/main/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStats.java

```diff
@@ -0,0 +1,66 @@
+package edu.asu.spring.quadriga.service.publicwebsite.impl;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.IConceptStats;
+import edu.asu.spring.quadriga.domain.impl.ConceptStats;
+import edu.asu.spring.quadriga.domain.network.INetwork;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.network.domain.ITransformedNetwork;
+import edu.asu.spring.quadriga.service.network.impl.INetworkTransformationManager;
+import edu.asu.spring.quadriga.service.publicwebsite.IProjectStats;
+import edu.asu.spring.quadriga.transform.Link;
+import edu.asu.spring.quadriga.transform.Node;
+
+/**
+ * This class represents business logic for getting top concepts from the given
+ * list of networks.
+ *
+ * @author ajaymodi
+ *
+ */
+
+@Service
+public class ProjectStats implements IProjectStats {
+
+	@Autowired
+	private INetworkTransformationManager transformationManager;
+
+	@Override
+	public List<IConceptStats> getTopConcepts(List<INetwork> Networks)

```

Comment:
variables should have lowercase names; please read the code style guide if you are unclear about java naming conventions